Breeze.BreezeServer/Program.cs

```diff
@@ -115,10 +115,21 @@ public static void Main(string[] args)
                 logger.LogInformation("{Time} Node registration has already been performed", DateTime.Now);
             }
 
+            // Perform collateral balance check and report the result
+            Money missingFunds;
+            if (registration.VerifyCollateral(config, out missingFunds))
+            {
+                logger.LogInformation($"{{Time}} The collateral address {config.TumblerEcdsaKeyAddress} has sufficient funds.", DateTime.Now);

```

Comment:
Perhaps add the computed balance to the log entry?

Reply:
The `missingFunds` will return 0 when the collateral balance is equal or grater than MASTERNODE_COLLATERAL_THRESHOLD.